test/helpers/deployer.js

```diff
@@ -412,10 +418,15 @@ class Deployer {
 
   async initializeChildChain(owner, options = { updateRegistry: true }) {
     // not mentioning the gas limit fails with "The contract code couldn't be stored, please check your gas limit." intermittently which is super weird
-    this.childChain = await contracts.ChildChain.new({ gas: 7500000 })
+    this.childChain = await contracts.ChildChain.new({ gas: 75000000 })
```

Comment:
I understand that you need more gas for coverage. But for normal tests does bor have enough gas limit? If not we can use something `gas: process.env.SOLIDITY_COVERAGE ? 75000000 : 7500000`.

Reply:
Side effect of messing around, no need to increase at all i think.